yew/src/virtual_dom/vtag.rs

```diff
@@ -1092,6 +1092,150 @@ mod tests {
             panic!("vtag expected");
         }
     }
+
+    #[test]
+    fn html_if_bool() {
+        assert_eq!(
+            html! {
+                if true {
+                    <div class="foo" />
+                }
+            },
+            html!(<div class="foo" />),
+        );
```

Comment:
These tests are in a test block which runs in the browser but they really don't need to. Can you move them to a normal test block?

Reply:
Sure. I had no idea. I will check how it is done

Reply:
I see. Done